pkgs/tools/graphics/obj2voxel/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, static ? stdenv.hostPlatform.isStatic
+, config
+, cmake
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "obj2voxel";
+  version = "1.3.4";
+
+  src = (
+    fetchFromGitHub {
+      owner = "Eisenwave";
+      repo = finalAttrs.pname;
+      rev = "refs/tags/v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-FkGQjYm15le1Na+2U1TgK4nNOZNs5bME5J87pjVFpDs=";
+    }
+  ).overrideAttrs (
+    _: {
+      GIT_CONFIG_COUNT = 1;
+      GIT_CONFIG_KEY_0 = "url.https://github.com/.insteadOf";
+      GIT_CONFIG_VALUE_0 = "git@github.com:";
+    }
+  );
```

Comment:
Interesting construction. I assume that your firewall has prevented from loading the submodules via ssh? I Dont know if this is the right way to circumvent this though.

Reply:
It's a workaround suggested here: https://github.com/NixOS/nixpkgs/issues/195117#issuecomment-1410398050

Not sure whether it's better to require SSH keys on build hosts.

Reply:
Ok, then lets keep it. Maybe it will be adapted in fetchgit itself one day.